tpm2_cmds.c

```diff
@@ -97,6 +97,11 @@ int tpm2_extend_pcr(struct tpm *t, u32 pcr,
 	u16 size;
 	int ret = 0;
 
+	if (b == NULL) {
```

Comment:
You should add this to tpm1_cmds.c too

Reply:
Will do! Will make it a separate commit.